vars/iqeUtils.groovy

```diff
@@ -206,7 +206,7 @@ def runIQE(String plugin, Map appOptions) {
     def marker = appOptions['marker']
     def extraArgs = appOptions['extraArgs']
 
-    catchError(stageResult: "ERROR") {
```

Comment:
I wasn't aware that this had to be one of a `tuple` of states. I added some detail about what I was trying to achieve here:
https://github.com/RedHatInsights/insights-pipeline-lib/pull/133/files#r608591478

My intent was that we shouldn't have Jenkins pipelines notifying of failures due to some pytest `InternalError` in the framework. Those messages should go to the error standup channel and this should be considered a state that is separate from `FAILURE`. If we want to leave this in (and I'd be ok just reverting #133 totally and revisiting it later), we could change the `stageResult` to something else.